tamr_client/mastering/project.py

```diff
@@ -9,3 +19,37 @@ def _from_json(url: URL, data: JsonDict) -> MasteringProject:
         data: Project JSON data from Tamr server
     """
     return MasteringProject(url, name=data["name"], description=data.get("description"))
+
+
+def create(
+    session: Session,
+    instance: Instance,
+    name: str,
+    description: Optional[str] = None,
+    external_id: Optional[str] = None,
+    unified_dataset_name: Optional[str] = None,
+) -> Project:
+    """Create a Mastering project in Tamr.
+    Args:
```

Comment:
Needs whitespace for doc formatting:

```suggestion

    Args:
```

Reply:
updated